internal/seclang/rule_parser.go

```diff
@@ -317,18 +329,23 @@ func ParseRule(options RuleOptions) (*corazawaf.Rule, error) {
 		rule:           corazawaf.NewRule(),
 		defaultActions: map[types.RulePhase][]ruleAction{},
 	}
-
-	defaultActions := options.Config.Get("rule_default_actions", []string{
-		defaultActionsPhase2,
-	}).([]string)
+	// Default actions are persisted only inside the waf options, therefore they are parsed every time a rule is parsed
```

Comment:
Another look may be needed, I'm a bit surprised about SecDefaultActions that are just "stored" and only parsed with ParseDefaultActions when a real rule is parsed (and ParseDefaultActions happens for **each** parsed rule)

Reply:
Agreed. I think we shouldn't need to read default actions if things are already defined in the rule. But this might have additional complexity...